main.cpp

```diff
@@ -31,6 +31,11 @@
 #pragma warning(disable : 4996)
 #endif
 
+#ifdef __clang__
+#pragma clang diagnostic ignored "-Wcovered-switch-default"
```

Comment:
In this case, `default` is not executed since all items are listed up in switch syntax. I have fixed it in this commit: https://github.com/KhronosGroup/SPIRV-Cross/pull/117/commits/5e5f93c2e7e1c0a521b97c2226244234839a9027#diff-118fcbaaba162ba17933c7893247df3aL350